Phonebook.Frontend/src/app/shared/dialogs/display-notification-dialog/display-notification.dialog.html

```diff
@@ -112,11 +112,22 @@ <h3 mat-dialog-title class="pb-color-primary" i18n="Display-notificationDialog|W
     <!-- call for contributors -->
     <app-contributors class="border-1 pb-center-element pb-border-color-primary pb-margin-05"></app-contributors>
   </mat-dialog-content>
+
   <div class="spacer"></div>
   <mat-dialog-actions>
-    <p i18n="Display-notificationDialog|Cookie notice@@WelcomeCookieNotice">
+    <span i18n="Display-notificationDialog|Cookie notice@@WelcomeCookieNotice">
       This site uses cookies to ensure the basic functionality of the app.
-    </p>
+    </span>
+    <span i18n="Display-notificationDialog|Annoyed by cookies notice@@PageInformationNewUrlNoCookiesText">
+      You deactivated cookies and keep getting those dialogs?
+    </span>
+    <button
+      i18n="Display-notificationDialog|Deactivate Startup Dialogues@@PageInformationNewUrlNoCookiesButton"
+      mat-button
+      (click)="skipDialoges()"
```

Comment:
rename to `skipDialogs()`

Reply:
Maybe we should find a general naming convention for that bacause @DanielHabenicht suggested "dialogues" in a previous comment. Of course you are right, "dialoges" is definitely not correct.

Reply:
than why not just `skipStartDialogs()` ? I think this is the best solution with best understanding

Reply:
Because the officially spelling is "dialogues". But of course we can agree on changing all occurrences to "dialogs".